modules/machine-pools-hcp.adoc

```diff
@@ -6,7 +6,7 @@
 [id="machine-pools-hcp_{context}"]
 = Machine pools in {hcp-title} clusters
 
-In {hcp-title} clusters, the hosted control plane spans three availability zones (AZ) in the installed cloud region. Each machine pool in a {hcp-title} cluster deploys in a single subnet within a single AZ. Each of these AZs can have only one machine pool. 
+In {hcp-title} clusters, the hosted control plane spans three availability zones (AZ) in the installed cloud region. Each machine pool in a {hcp-title} cluster deploys in a single subnet within a single AZ. Each of these AZs can have only one private subnet. 
```

Comment:
```suggestion
In {hcp-title} clusters, the hosted control plane spans three availability zones (AZ) in the installed cloud region. Each machine pool in a {hcp-title} cluster deploys in a single subnet within a single AZ. Each of these AZs can have only one private subnet for machinepools.
```
Sorry, I don't think "one private subnet" is correct either, customers can have as many private subnets as they want, as long as only one of them per AZ is used for machinepools